### ISSUE TITLE
Postgres 10

### ISSUE THREAD

**Issue by weymanf** [cloudfoundry/postgres-release#39]
Hello, I didn't know where I should ask about this. I just wanted to know if there were plans to use postgres 10 in this release?

**Comment by valeriap** [comment]
@weymanf 
At the moment it was not in our plan but we added a story to the icebox for it.
Do you have specific needs that requires this major upgrade?

**Comment by weymanf** [comment]
Yes. We need the new partitioning syntax, basic overall performance, and improved query planning.

If there is a timeline that is in your mind can you please share it? I have a fork with postgres 10 but I don't really want to use it because of long going support.

**Comment by valeriap** [comment]
We have opened a story to investigate the impact of changes in PostgreSQL 10.x. See [story](https://www.pivotaltracker.com/story/show/155584920).
Please see the [tracker](https://www.pivotaltracker.com/story/show/155584920) for details.

**Comment by jchesterpivotal** [comment]
PostgreSQL 11 has been released as well, with additional improvements for partitioning and parallel queries.

**Comment by valeriap** [comment]
@jchesterpivotal  The PostgreSQL major upgrade is the next story in the backlog and the plan is to jump directly to version 11.

**Comment by jchesterpivotal** [comment]
I love everything about this news :)

**Comment by alexwo** [comment]
Hi @valeriap,

Thats are great news 👍  

Do you know if the cf-deployment, bosh or maybe concourse projects have corresponding backlog to validate that they can perform well on top of Postgres11?  (  Or maybe similar efforts were done once version 9.6.10 was enabled? )

Thanks,
Alex

**Comment by valeriap** [comment]
@alexwo 
We already run core CATS tests for cf-deployment in our pipeline.
Bosh is not impacted by our change because they maintain their own PostgreSQL package.
